EIPS/eip-9999.md

```diff
@@ -0,0 +1,84 @@
+---
+eip: 
+title: Anti-correlation attestation penalties
+description: Adjust penalties for missed attestations based on in-slot correlation of missed attestation
+author: dapplion (@dapplion), Toni Wahrstätter (@nero_eth), Vitalik Buterin (@vbuterin)
+discussions-to: 
```

Comment:
Please create a topic in Eth Magicians for this EIP with a link to this PR
https://ethereum-magicians.org/